packages/opentelemetry-cloud-trace-exporter/package.json

```diff
@@ -10,11 +10,12 @@
     "lint": "gts lint",
     "clean": "gts clean",
     "precompile": "version-update.js",
-    "compile": "tsc",
+    "compile": "tsc && cp -r protos build",
     "prepack": "npm run compile",
     "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
     "fix": "gts fix",
-    "pretest": "npm run compile"
+    "pretest": "npm run compile",
+    "compile-protos": "node compile-protos.js"
```

Comment:
Can you add this to run during `precompile` step?

Reply:
I tested a few scenarios, and it turns out that if we move the JSON file directly into the build directory and remove `/protos/protos.json`, the reference from `trace.ts` results in an error.

However, during my testing, I realized that we don’t need to run `cp -r protos build` because the JSON file is already included in the build due to the include option in the tsconfig.json. Therefore, I have removed that command since it was redundant. → [1558ee4](https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/pull/741/commits/1558ee4d3b7cff4e214b621dd3e1adccef8585b4)

If you know of any alternative ways to handle this without removing `protos/protos.json`, I’d appreciate your idea!

Reply:
I guess we could simply add it to .gitiginore.

However I think we what have here is fine–I doubt the generated code will change often anyway. Thank you for looking into it.